src/test/scala/ECommAlgorithmTest2.scala

```diff
@@ -0,0 +1,408 @@
+package org.template.ecommercerecommendation
+
+import org.scalatest.FlatSpec
+import org.scalatest.Matchers
+
+import io.prediction.data.storage.BiMap
+
+import org.apache.spark.mllib.recommendation.ALS
+import org.apache.spark.mllib.recommendation.{Rating => MLlibRating}
+
+import org.scalatest.junit.JUnitRunner
+import org.junit.runner.RunWith
+
+@RunWith(classOf[JUnitRunner])
+class ECommAlgorithmTest2
```

Comment:
Renamed class to ECommAlgorithmTestTraining.scala as it tests two training methods of ECommAlgorithm class. Removed test method predictKnownUser since the test in ECommAlgorithmTest.scala of this method is sufficient